src/deploy-tes-on-azure/Configuration.cs

```diff
@@ -83,6 +83,7 @@ public abstract class UserAccessibleConfiguration
         public string PostgreSqlServerName { get; set; }
         public bool UsePostgreSqlSingleServer { get; set; } = false;
         public string KeyVaultName { get; set; }
+        public string AzureName { get; set; } = "AzureGlobalCloud";
```

Comment:
rename to "AzureEnvironmentName", which I think makes more sense right?  Also can we use an enumeration here instead of a string?

Reply:
Considering 'AzureEnironmentName' is too long, I prefer keep AzureName as parameter name.